packages/@react-types/combobox/src/index.d.ts

```diff
@@ -34,16 +35,29 @@ export interface ComboBoxProps<T> extends CollectionBase<T>, SingleSelection, In
   //  * @default 'suggest'
   //  */
   // completionMode?: 'suggest' | 'complete',
-  /**
-   * The interaction required to display the ComboBox menu.
-   * @default 'input'
-   */
-  menuTrigger?: MenuTriggerAction,
+ /**
+  * The interaction required to display the ComboBox menu.
+  * @default 'input'
+  */
+  menuTrigger?: MenuTriggerAction
+}
+
+export interface AriaComboBoxProps<T> extends ComboBoxProps<T>, DOMProps {
+  /** The ref for the input element. */
+  inputRef: RefObject<HTMLInputElement>,
+  /** The ref for the list box popover. */
+  popoverRef: RefObject<HTMLDivElement>,
+  /** The ref for the list box. */
+  listBoxRef: RefObject<HTMLElement>,
+  /** The ref for the optional list box popup trigger button.  */
+  buttonRef?: RefObject<HTMLElement>,
+  /** An optional keyboard delegate implementation, to override the default. */
+  keyboardDelegate?: KeyboardDelegate,
   /** Whether keyboard navigation is circular. */
   shouldFocusWrap?: boolean
 }
 
-export interface SpectrumComboBoxProps<T> extends Omit<ComboBoxProps<T>, 'menuTrigger'>, SpectrumLabelableProps, StyleProps, Omit<AsyncLoadable, 'isLoading'> {
+export interface SpectrumComboBoxProps<T> extends AriaComboBoxProps<T>, SpectrumLabelableProps, StyleProps, Omit<AsyncLoadable, 'isLoading'> {
```

Comment:
sorry, I slightly forgot the Omit syntax in my earlier review, I think this should now be, which you might've already had, if so, sorry about the confusion
```suggestion
export interface SpectrumComboBoxProps<T> extends Omit<AriaComboBoxProps<T>, 'menuTrigger'>, SpectrumLabelableProps, StyleProps, Omit<AsyncLoadable, 'isLoading'> {
```

Reply:
👍